packages/react-scripts/template/src/components/example/ArtifactsViewer.js

```diff
@@ -8,15 +8,31 @@ import Banner from './Banner'
 const artifactsCss = css`
   margin: 0 -24px;
 `
+const artifactsViewSkeletonCss = css`
+  margin: 0 -24px;
+  display: flex;
+  & > div {
+    margin-right: 8px;
+  }
```

Comment:
will this be typical CSS that teams will need to use with skeleton, or are we doing something slightly unusual with artifacts viewer?

Reply:
Little bit different. And custom.